specification/cognitiveservices/data-plane/ComputerVision/v1.0/ComputerVision.json

```diff
@@ -0,0 +1,1468 @@
+                    }
+                }
+            }
+        },
+        "ComputerVisionError": {
+            "type": "object",
+            "required": [
+                "code",
+                "message"
+            ],
+            "properties": {
+                "code": {
+                    "type": "string",
+                    "description": "The error code.",
+                    "enum": [
+                        "InvalidImageUrl",
+                        "InvalidImageFormat",
+                        "InvalidImageSize",
+                        "NotSupportedVisualFeature",
+                        "NotSupportedImage",
+                        "InvalidDetails",
+                        "NotSupportedLanguage",
+                        "BadArgument",
+                        "FailedToProcess",
+                        "Timeout",
+                        "InternalServerError",
+                        "Unspecified",
+                        "StorageException"
+                    ],
+                    "x-ms-enum": {
+                        "name": "ComputerVisionErrorCodes",
+                        "modelAsString": false
+                    }
+                },
+                "message": {
+                    "type": "string",
+                    "description": "A message explaining the error reported by the service."
+                },
+                "requestId": {
+                    "type": "string",
+                    "description": "A unique request identifier."
+                }
+            }
+        }
+    },
+    "parameters": {
+        "VisualFeatures": {
+            "name": "visualFeatures",
+            "in": "query",
+            "description": "A string indicating what visual feature types to return. Multiple values should be comma-separated. Valid visual feature types include:Categories - categorizes image content according to a taxonomy defined in documentation. Tags - tags the image with a detailed list of words related to the image content. Description - describes the image content with a complete English sentence. Faces - detects if faces are present. If present, generate coordinates, gender and age. ImageType - detects if image is clipart or a line drawing. Color - determines the accent color, dominant color, and whether an image is black&white.Adult - detects if the image is pornographic in nature (depicts nudity or a sex act).  Sexually suggestive content is also detected.",
+            "type": "array",
+            "x-ms-parameter-location": "method",
+            "required": false,
+            "collectionFormat": "csv",
+            "items": {
+                "type": "string",
+                "x-nullable": "false",
+                "x-ms-enum": {
+                    "name": "VisualFeatureTypes",
+                    "modelAsString": false
+                },
+                "enum": [
+                    "ImageType",
+                    "Faces",
+       
```

Comment:
we don't need an enum for boolean types

Reply:
Did you mean to highlight "HandwritingBoolean"? I removed the enum from that.